pyglidein/startd_cron_scripts/clsim_gpu_test.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import os
+from optparse import OptionParser
+from subprocess import check_output, STDOUT
+import xml.etree.ElementTree as ET
+
+OUTPUT_FILE = 'benchmark.xml'
+
+CLASSAD_MAP = {'I3CLSimModule_makeCLSimHits_makePhotons_clsim_AverageDeviceTimePerPhoton':
+               'PYGLIDEIN_METRIC_TIME_PER_PHOTON'}
+
+
+def main():
+
+    usage = "usage: %prog [options]"
+    parser = OptionParser(usage)
+    parser.add_option('-n', type='int', default=1,
+                      help="Number of simulations to run")
+    (options, args) = parser.parse_args()
+
+    try:
+        # This line is needed to access the nvidia gpus via OpenCL inside a container.
+        # TODO: Is this needed in a grid environment?
+        os.environ['LD_LIBRARY_PATH'] = ('/usr/local/nvidia/lib64:'
+                                         '{}'.format(os.environ['LD_LIBRARY_PATH']))
```

Comment:
`os.environ[]` should already return a string.  is there a reason to use `.format()` here, or would something like this be better?
```python
        os.environ['LD_LIBRARY_PATH'] = ('/usr/local/nvidia/lib64:' + os.environ['LD_LIBRARY_PATH'])
```

Also, I should note that we might need to check that the key exists in `os.environ` before blindly using.

Reply:
Fixed.